app/data_model/session_data.py

```diff
@@ -1,6 +1,6 @@
 class SessionData(object):
 
-    def __init__(self, tx_id, eq_id, form_type, period_str, language_code=None, survey_url=None, ru_name=None, ru_ref=None, submitted_time=None):
+    def __init__(self, tx_id, eq_id, form_type, period_str, language_code=None, survey_url=None, ru_name=None, ru_ref=None, submitted_time=None, **kwargs):
```

Comment:
For everything other than **submitted_time** we should really remove the None defaults now. These were only needed for backward compatibility.